projects/plugins/boost/readme.txt

```diff
@@ -9,39 +9,45 @@ Stable tag: 1.4.2
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
+== Description ==
+
 Speed up your website by optimizing page performance with Jetpack Boost!
 
-== Description ==
+Did you know that a faster website:-
+
+- Ranks higher on Google.
+- Improves bounce rate (people stay on your site for longer).
+- Increases your conversion rate.
+
+Increase your website performance and speed up your website with one-click optimizations that supercharge your WordPress site’s performance and improve web vitals scores for better search engine listings.
 
-Jetpack Boost provides one-click optimizations that supercharge your WordPress site’s performance and improve web vitals scores for better SEO.
+Improving Core Web Vitals helps you rank higher on Google. A faster website also improves your SEO, helps you reduce your bounce rate and increase your ecommerce conversion rate. 
+
+- Largest Contenful Paint (LCP): Measures loading performance. Improve your LCP and improve your website loading speed.
+- First Input Delay (FID): Measures interactivity. To improve user experience pages should have a low FID.
+- Cumilative Layout Shift (CLS): Measures visual stability. Lowering your CLS helps improve your user experience.
```

Comment:
These three bullet points seem very useful to include in the product description, as these are web metrics that our tools can improve. But we currently don't display them anywhere in the app's dashboard. Is that going to cause confusion?

Reply:
> But we currently don't display them anywhere in the app's dashboard. Is that going to cause confusion?

Good Question. I don't think they'll cause confusion? At the minimum we could add a "Did you know" with a link to here: https://pagespeed.web.dev/ to the UI if we want to have a resource in there?

But, I don't think that's critical (since we mention web vitals already in the readme, but don't display them anywhere)